source/manual/set-up-aws-account.html.md

```diff
@@ -0,0 +1,62 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Set up your AWS account
+section: AWS accounts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-09-03
+review_in: 6 months
+redirects:
+  - /manual/user-management-in-aws.html
+---
+
+To work with [govuk-aws](https://github.com/alphagov/govuk-aws) and [govuk-aws-data](https://github.com/alphagov/govuk-aws-data),
+you will require an account in AWS.
+
+## 1. Request a GDS AWS account
+
+GDS maintains a central account for AWS access. You will need to request an account from the Technology and Operations team.
+
+👉 [Request an account](https://gds-request-an-aws-account.cloudapps.digital)
+
+## 2. Sign in to AWS
+
+To sign in, go to [the GDS AWS Sign page](https://gds-users.signin.aws.amazon.com/console), and use the following credentials:
+
+- "Account ID or alias": `gds-users`
+- Username: your `@digital.cabinet-office.gov.uk` email address
+- Password: your password
+
+👉 [Sign in to AWS GDS account](https://gds-users.signin.aws.amazon.com/console)
+
+## 3. Set up your MFA
+
+You have to set up [Multi-Factor Authentication (MFA)][MFA].
+
+## 4. Get the appropriate access
+
+An account in AWS doesn't give you access to anything, you'll need to be given rights.
+
+Add yourself to the lists of users found in [the data for the infra-security project][infra-terra].
```

Comment:
@rubenarakelyan do we document somewhere who should have what role?

Reply:
I don't know if it's written down anywhere, but the rules are basically:

* `govuk-administrators`: People in RE who are working on GOV.UK infrastructure, senior tech members of GOV.UK and anyone else working on the AWS migration
*  `govuk-powerusers`: Anyone else who has production access on GOV.UK
* `govuk-users`: Anyone else who has integration access on GOV.UK

Reply:
Thanks! Added.